test/end-to-end/util/helpers.ts

```diff
@@ -132,3 +132,14 @@ export const copyAlert = Selector('div').withText('Email has been copied')
 
 // Transition Page
 export const skipButton = Selector('#skip')
+
+// URL Drawer
+export const urlDisableToggleButton = Selector('input[type="checkbox"]')
```

Comment:
we can use the `activeSwitch` defined above for this!

Reply:
thank you for this!